libs/coin-modules/coin-xrp/src/api/index.ts

```diff
@@ -162,15 +164,15 @@ async function operationsFromHeight(
 
 // NOTE: double check
 async function operations(address: string, pagination: Pagination): Promise<[Operation[], string]> {
-  const { minHeight, lastPagingToken } = pagination;
+  const { minHeight, lastPagingToken, order } = pagination;
   if (minHeight) {
-    return await operationsFromHeight(address, minHeight);
+    return await operationsFromHeight(address, minHeight, order);
   }
   const isInitSync = lastPagingToken === "";
 
   const newPagination = {
     minHeight: isInitSync ? 0 : parseInt(lastPagingToken || "0", 10),
   };
   // TODO token must be implemented properly (waiting ack from the design document)
-  return await operationsFromHeight(address, newPagination.minHeight);
+  return await operationsFromHeight(address, newPagination.minHeight, order);
```

Comment:
same why not remove the operationsFromHeight and call logic.listOperations and use the token ?